pygmt/src/histogram.py

```diff
@@ -10,16 +10,29 @@
     A="horizontal",
     B="frame",
     C="cmap",
+    D="annotate",
+    E="barwidth",
+    F="center",
     G="fill",
     J="projection",
+    L="extreme",
+    N="normal",
```

Comment:
Is `N="normal"` a good alias? [GMT.jl uses `distribution`](https://www.generic-mapping-tools.org/GMT.jl/dev/histogram/). The `normal` alias sounds like the histogram is normalized to 1 and may be confusing?

Reply:
Switched it to `distribution`. I feel indifferent about which name to use, but I think it should be standardized with the other GMT wrappers.